### ISSUE TITLE
Make it easier to join Slack

### ISSUE THREAD

**Issue by davisshaver** [newsnerdery/newsnerdery#4]
Let's make Daniel's life easier by adding a button to signup folks to Slack. We could setup [Slack.in](http://rauchg.com/slackin/) for this. [See here for example.](http://phillydev.org/)

**Comment by danielbachhuber** [comment]
Good suggestion. It doesn't appear to have moderation functionality, but I've opened an issue about it: https://github.com/rauchg/slackin/issues/53

**Comment by dannguyen** [comment]
Is there a current need to have moderation of who can join? I'm asking that as a Slack beginner, if using slack.in results in userspambots. I noticed that [chat.18f.gov](https://chat.18f.gov/) uses slack.in. It'd be great to be able to direct people to a link to join...not just for them but to save the mods some manual work.

**Comment by davisshaver** [comment]
@dannguyen Good question. I have no reason to suspect spam. So another reason to ask this: Is there any one we would _reject_ at this point?

**Comment by danielbachhuber** [comment]
> Is there a current need to have moderation of who can join?

I like moderation for the time being.

**Comment by jtannenwald** [comment]
I don't trust spambots to stay away?

**Comment by davisshaver** [comment]
@daniel can you expand on _why_? I think that requiring people _ask_ to join is a bit exclusive, even though our mods are a diverse bunch.

**Comment by danielbachhuber** [comment]
> I think that requiring people ask to join is a bit exclusive, even though our mods are a diverse bunch.

That's the intention — people like the feeling of being a part of a club. The feeling of exclusivity is fine when pragmatically you're inclusive.

**Comment by davisshaver** [comment]
If we're just trying to _act_ exclusive, I don't see the upside at all. There are going to be people turned away from the pretension of exclusivity if they expect that they wouldn't make the cut. I'm going on record against this policy as I think it's counter-productive to having a diverse news nerdery.

**Comment by danielbachhuber** [comment]
> I'm going on record against this policy as I think it's counter-productive to having a diverse news nerdery.

That's fine.

**Comment by juruwolfe** [comment]
Yeah, I'd like to second Davis's point. I do think the appearance of exclusivity, even if we don't mean it to, keeps certain people/voices at bay. Not sure that cost is worth potentially being _more_ alluring for voices that want to be part of the cool club.
